components/dealInDetails/sliderAndTimerComponents/DealAndStoreDetail.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Button, Divider, Grid, Typography, colors } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import { all_center } from "../../../constant/commonStyle";
+import { useSelector } from "react-redux";
+import theme from "../../../theme";
+import GetDealsAndClicks from "./GetDealsAndClicks";
+import LimitedOffer from "./LimitedOffer";
+import Store from "./Store";
+import ApplicableLocations from "./ApplicableLocations";
+
+
+
+
+
+
+const DealAndStoreDetail = () => {
+  const [timer, setTimer] = useState(true);
```

Comment:
Where you have used `timer`?

Reply:
Sir there is no use of timer i will remove it.